### PR TITLE
Gauravbalade myartwork

### PR DESCRIPTION
**Pull request by gauravbalade**
What I did
In this pull request, I created a new directory under my GitHub username gauravbalade titled gauravbalade-myartwork. This directory contains an HTML and CSS artwork for Hacktoberfest 2024. The submitted files include index.html, styles.css, and meta.json.

Pull request checklist:
[x] This pull request uses HTML and CSS only and provides a meta.json file.

[x] All files in this pull request are in a directory named: gauravbalade-myartwork without the <>.

[x] My pull request contains index.html and styles.css.

[x] My html file is named exactly: index.html.

[x] My css file is named exactly: styles.css.

[x] My pull request includes a meta.json with:

[x] githubHandle: being my unique GitHub username, gauravbalade

[x] artName: name of my animation, "Contribute with Style"

[x] My submission includes at least one animation (blink, fadeIn, slideIn, bounce).

[x] My pull request does not modify any other files in the repository.

Additional Information:
The index.html file contains a welcoming message and descriptions for the Hacktoberfest project.

The styles.css file includes various CSS animations like blink, fadeIn, slideIn, and bounce to enhance the artwork visually.

The meta.json file provides detailed metadata about the artwork, including descriptions, tags, version, license, and contributor information.

I ensured all the criteria are met to adhere to the repository's guidelines. Looking forward to contributing and enhancing the Hacktoberfest project with my submission!@zero-to-mastery #3 #hacktoberfest #hacktoberfest-accepted #gauravbalade

### CODE REVIEW

**Comment by gauravbalade** [comment]
_Thankyou for this Resp and hence like to collaborate with it._ @zero-to-mastery/gitgang @gauravbalade @zero-to-mastery/zerotomastery-contributors  @zero-to-mastery @l-white @hacktoberfest

File: Art/gauravbalade-myartwork/image.png
Comment (LaurelineP) [inline Art/gauravbalade-myartwork/image.png:1]: # Unecessary picture
Please remove the file - an icon will be generated : there is no need to add the picture.

File: Art/gauravbalade-myartwork/meta.json
Comment (LaurelineP) [inline Art/gauravbalade-myartwork/meta.json:1]: ## Invalid `meta.json`
Please check the code's README.md file
The content should only have 2 fields
- githubHandle
- artName

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @gauravbalade  and thanks for your contribution 🙌
In order for us to merge your PR please address the provided feedbacks :)

File: Art/gauravbalade-myartwork/image.png
Comment (gauravbalade) [reply Art/gauravbalade-myartwork/image.png:1]: Thank you, LaurelineP, for your feedback. I have removed the unnecessary image as requested. This should align perfectly with the project guidelines. I appreciate the opportunity to contribute and refine my submission.

Please review the changes at your convenience. Looking forward to your approval!

Best regards, Gaurav Balade

**Pull request merged**